guides/V1_space_template_main.md

```diff
@@ -0,0 +1,21 @@
+---
+copyright: 'Copyright IBM Corp. 2018'
+link: 'space-template-main'
+is: 'experimental'
+---
+
+# Space Template
+
+In Watson Work Services a space template is a template for the creation of a space.
+The space template defines user configurable properties for the space, 
+as well as a set of required apps that are automatically added to the space on creation.
+
+# Key fields
+
+| property      | type          | description  |
+| ------------- |:------------- |:-----|
+| properties |SpacePropertyCollection |There are three types of properties in a space template: List, Boolean, and Text.|
+| status |SpaceStatus |Status is a special list style attribute on the template. Status consists of a list of values that can be used to describe the state of the space; for example, open, closed, archived.|
+| requiredApps |SpaceRequiredApplicationCollection |Required apps define a set of applications (by ID) that are added to each new space on creation.|
+| teamId |String |The id of the team in which the space will be created, derived either from the offering with which the template is associated (in the case of an offering template) or from the user who created the template (in the case of a user-created custom template)|
```

Comment:
>derived either from the offering with which the template is associated (in the case of an offering template)

It is really derived from the team that the user who is creating the space belongs to no?  That user's team is what is associated to an offering... which is what gave the user access to the template.

Reply:
Probably would be good to specify why the team is to be derived that way vs it being simply the team of the caller/user.

Reply:
IN the case of offering templates the template is linked to an offering, and the user has permission to use these templates if they have that offering at the time. In the case of user created templates the user can create a space in whatever offering they are in, and we just default to the first offering we find for now.

Reply:
>In the case of user created templates the user can create a space in whatever offering they are in, and we just default to the first offering we find for now.

Wondering if this type of info should go into a "concepts/introduction" type guide for templates.  Thoughts?

Reply:
We will be expanding the space-template-main page as the "concepts/introduction" type guide. So it's in the right place for now. 😄